src/components/Home/Home.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useContext } from 'react';
+
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import { withStyles } from '@material-ui/core/styles';
+import styles from './Home.styles';
+import { documentTitle } from '../../common/variables';
+import Menu from '../common/Menu';
+import { AppContext } from '../../context/AppContext';
+import { hasPermission, POLICIES } from '../../models/auth';
+import {
+  DashStatGrowerCount,
+  DashStatTotalCaptures,
+  DashStatUnprocessedCaptures,
+  DashStatVerifiedCaptures,
+} from '../DashStat.container';
+import GreenStandSvgLogo from '../common/GreenStandSvgLogo';
+
+/**
+ * @function
+ * @name Home
+ * @description renders the home page
+ * @param {object} props
+ * @param {object} props.classes css classes
+ *
+ * @returns {React.Component} Home component
+ */
+function Home(props) {
+  const { classes } = props;
+  const appContext = useContext(AppContext);
+
+  useEffect(() => {
+    document.title = `${documentTitle}`; // update page title
```

Comment:
That's an interesting suggestion. How would you distinguish _leftover_ comments from legitimate, useful comments?

Reply:
We could set up es lint to lint for comments that would prevent pushing with comments, and then when there is a need for a useful comment you can explicitly tell es lint to ignore the no comment rule for that line or file.

It would just help keep things clean by preventing unnecessary comments, left over comments to oneself, and code that is commented out while working that is no longer needed but people often forget to remove.

Reply:
I like the sentiment, but I'm concerned that it would discourage (particularly new) contributors from commenting.
I'd rather have too many comments than not enough.

Clearing up commented out code is a fairly straightforward activity; we could make it part of the code review and encourage developers to remove unused, commented-out code as they go along as standard code maintenance practice.

Reply:
That makes sense to me - leaving it to code review and building habits that way works too!